Lib/ufo2ft/__init__.py

```diff
@@ -532,42 +532,83 @@ def compileVariableTTF(designSpaceDoc, **kwargs):
     The rest of the arguments works the same as in the other compile functions.
 
     Returns a new variable TTFont object.
+
+    .. deprecated:: TODO (Jany) version of deprecation
```

Comment:
i don't see why it should be deprecated, it's perfectly fine to be wanting to only build one vf at a time, if you know sources only have that

Reply:
My thinking was that it doesn't bring any value over the plural function, but it will be an extra function to maintain

Reply:
as long as it's implemented in terms of the other one, I don't think it's an extra burden. Having to change one's client code to avoid the deprecation warning is extra burden